src/SwaggerProvider/SwaggerProvider.fsproj

```diff
@@ -10,10 +10,11 @@
     <RootNamespace>SwaggerProvider</RootNamespace>
     <AssemblyName>SwaggerProvider</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
+    <TargetProfile>netcore</TargetProfile>
+    <TargetFSharpCoreVersion>3.1.2.5</TargetFSharpCoreVersion>
```

Comment:
Template in `generator-fsharp` also have all these lines... https://github.com/fsprojects/generator-fsharp/pull/25/files 

Does it incorrect?